src/pcapi/admin/custom_views/offer_view.py

```diff
@@ -70,3 +78,67 @@ def _get_venue_name(self) -> str:
             abort(404, "Ce lieu n'existe pas ou plus")
 
         return venue.name
+
+
+def _offer_pc_links(view, context, model, name) -> Markup:
+    url = settings.PRO_URL + f"/offres/{humanize(model.id)}/edition"
+    text = "Offre PC"
+
+    return Markup(f'<a href="{url}" target="_blank" rel="noopener noreferrer">{text}</a>')
+
+
+def _related_offers_links(view, context, model, name) -> Markup:
+    url = url_for("offer_for_venue.index", id=model.venue.id)
+    text = "Offres associées"
+
+    return Markup(f'<a href="{url}">{text}</a>')
+
+
+def _metabase_offers_links(view, context, model, name) -> Markup:
+    if IS_PROD or IS_STAGING:
+        url = (
+            "https://data-analytics.internal-passculture.app/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJx"
+            "dWVyeSI6eyJzb3VyY2UtdGFibGUiOjEyfSwidHlwZSI6InF1ZXJ5In0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2"
+            "V0dGluZ3MiOnt9fQ=="
+        )
+        text = "Offre"
+
+        return Markup(f'<a href="{url}" target="_blank" rel="noopener noreferrer">{text}</a>')
+    return Markup("")
+
+
+class ValidationView(BaseAdminView):
+    can_create = False
+    can_edit = True
+    can_delete = False
+    column_list = ["id", "name", "validation", "venue.name", "score", "offer", "offers", "metabase"]
+    column_sortable_list = ["id", "name", "validation"]
+    column_labels = {
+        "name": "Nom",
+        "type": "Type",
+        "validation": "Validation",
+        "venue.name": "Lieu",
+        "offer": "Offre",
+        "score": "Score",
+        "metabase": "Metabase",
+    }
+    column_filters = ["type"]
+    form_columns = ["validation"]
+    simple_list_pager = True
+
+    @property
+    def column_formatters(self):
+        formatters = super().column_formatters
+        formatters.update(offer=_offer_pc_links)
+        formatters.update(offers=_related_offers_links)
+        formatters.update(metabase=_metabase_offers_links)
+        return formatters
+
+    def on_model_change(self, form: Form, offer: Offer, is_created: bool = False) -> None:
+        redis.add_offer_id(client=app.redis_client, offer_id=offer.id)
```

Comment:
Cela devrait être conditionné par le _feature flag_ `SYNCHRONIZE_ALGOLIA` :

```suggestion
        if feature_queries.is_active(FeatureToggle.SYNCHRONIZE_ALGOLIA):
            redis.add_offer_id(client=app.redis_client, offer_id=offer.id)
```